elastic/apm-server
# Ensure integration package validity using elastic-package tool

## Motivation/summary

Ensure the integration package is valid by running `elastic-package check` as part of `make check`.

Add Make targets:
 - make check-package
 - make format-package (run by `make gen-package`)
 - make build-package (run by `make gen-package`)

Some minor formatting was needed, as well as adding a link to the changelog entry.

Later we should consider getting rid of our custom README.template rendering, and rely on `elastic-package build` to render a template located at `_dev/build/docs/README.template`, like in integrations.

## How to test these changes

Mess up the formatting of the package, and run `make check-package`. It should fail and tell you the package is not formatted correctly.

## Related issues

Closes #4930
